webapp/src/views/projects/dashboard/ProjectTotals.tsx

```diff
@@ -0,0 +1,175 @@
+              {t('project_dashboard_key_count', 'Keys', {
+                count: stats.keyCount,
+              })}
+            </TileDescription>
+          </TileDataItem>
+          <TileDataItem>
+            <TileValue fontSize={28}>{stats.baseWordsCount}</TileValue>
+            <TileDescription>
+              {t('project_dashboard_base_words_count', 'Base words', {
+                count: stats.baseWordsCount,
+              })}
+            </TileDescription>
+          </TileDataItem>
+        </DoubleTile>
+
+        <DoubleTile>
+          <TileDataItem>
+            <TileValue fontSize={28}>
+              {!isNaN(stats.translatedPercentage)
+                ? t(
+                    'project_dashboard_percent_count',
+                    '{percentage, number, :: % }',
+                    {
+                      percentage: stats.translatedPercentage / 100,
+                    }
+                  )
+                : t('project_dashboard_percent_nan', '-')}
+            </TileValue>
+            <TileDescription>
+              {t('project_dashboard_translated_percent', 'Translated')}
+            </TileDescription>
+          </TileDataItem>
+          <TileDataItem>
+            <TileValue fontSize={28}>
+              {!isNaN(stats.reviewedPercentage)
+                ? t(
+                    'project_dashboard_percent_count',
+                    '{percentage, number, :: % }',
+                    {
+                      percentage: stats.reviewedPercentage / 100,
+                    }
+                  )
+                : t('project_dashboard_percent_nan', '-')}
+            </TileValue>
+            <TileDescription>
+              {t('project_dashboard_reviewed_percent', 'Reviewed')}
+            </TileDescription>
+          </TileDataItem>
+        </DoubleTile>
+
+        <DoubleTile>
+          <TileDataItem>
+            <Box>
+              <OwnerAvatar
+                userOwner={project.userOwner}
+                organizationOwner={project.organizationOwner}
+              />
+            </Box>
+            <TileDescription>
+              {project.userOwner?.name || project.organizationOwner?.name}
+            </TileDescription>
+            <TileSubDescription>
+              {t('project_dashboard_project_owner', 'Project Owner')}
+            </TileSubDescription>
+          </TileDataItem>
+          <TileDataItem>
+            <TileValue fontSize={28}>{stats.membersCount}</TileValue>
+            <TileDescription>
+              {t('project_dashboard_member_count', 'Members')}
+            </TileDescription>
+          </TileDataItem>
+        </DoubleTile>
+
+        <Tile>
+          <TileDataItem>
+            <TileValue>{stats.tagCount}</TileValue>
+            <TileDescription>
+              {t('project_dashboard_tag_count', 'Tags')}
+            </TileDescription>
+          </TileDataItem>
+        </Tile>
+      </Tiles>
+    </>
+  );
+};
+
+const Tiles = styled(Box)`
```

Comment:
Missing `Styled` prefix. I don't persist on that, but it's already everywhere, so I would do it like that.

Reply:
Right.